index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
+<html lang="en">
+<head>
+
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="Cindy Kandie is a well equipped frontend developer who has taken time to polish her craft. This Portfolio is a visual representation of the work she has done and what she can do." />
+    <meta name="keywords" content="frontend developer, web developer" />
+    
+
+    <link rel="stylesheet" href="styles.css">
+    <link rel="shortcut icon" href="assets/favicon.ico" type="image/x-icon">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=BhuTuka+Expanded+One&family=Poppins:wght@200;400;500;600;700;800;900&display=swap" rel="stylesheet">
+
+    <title>Cindy Kandie's Web Development Portfolio</title>
+</head>
+<body>
+    <section class="intro_section">
+        <header>
+            <div class="logo">
+                <p>Cindy K</p>
+            </div>
+            <nav>
+                <img src="assets/menu.png" alt="navbar">
+            </nav>
+        </header>
+        <article>
+            <h1>I’m Cindy <br>
+                Glad to see you!
+            </h1>
+            <p>I’m a Frontend developer! I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+            </p>
+        </article>
+        <div class="contact-me">
+            <h2>Let's connect</h2>
+            <div class="socials">
+                <img src="assets/twitter.svg" alt="My twitter page" >
+                <img src="assets/linkedin.png" alt="My linkedin page" >
+                <img src="assets/medium.svg" alt="My Medium page" >
+                <img src="assets/github.svg" alt="My GitHub page" >
+                <img src="assets/chill.svg" alt="My Peace page" >
```

Comment:
- [ ] Since it's going to be linked to the respective social media. It would be better to `a` tag instead of `img`.